### PR TITLE
merge and update chat 

### PR DESCRIPTION
**Pull request by leeannachen**
image

### CODE REVIEW

File: proj/src/main/webapp/index.html [hunk 1]
```diff
@@ -28,14 +28,12 @@
...
-        <input type="text" id="message-input" />
-        <button type="button" onclick="pushChatMessage()" id="send-button">send</button>
-        <div id="content"></div>
-        <div id="messages"></div>
         <a href="/static/welcome.html">Welcome Page</a>
+        <a href="/static/chat.html">Chat Page</a>
 
+        <!-- <ul id="chat-as-list"></ul>
+        <input type="text" id="message-input" />
+        <button type="button" onclick="pushChatMessage()" id="send-button">send</button> -->
```
Comment (aokeke) [inline proj/src/main/webapp/index.html]: Can you remove this commented out bit if it isn't being used.

File: public/script.js [hunk 2]
```diff
@@ -1,32 +0,0 @@
...
-    });
-}
-
-function pushChatMessage() {
-    const messageRef = firebase.database().ref(path);
-
-    const message = document.getElementById('message-input').value;
-    // push message to datastore
-    messageRef.push(message);
-}
```
Comment (aokeke) [inline public/script.js:32]: Where all these files intentionally deleted?

**Review by aokeke — CHANGES_REQUESTED** [review]

File: proj/src/main/webapp/index.html (hunk 1 above)
Comment (leeannachen) [reply proj/src/main/webapp/index.html]: Just removed it

File: public/script.js (hunk 2 above)
Comment (leeannachen) [reply public/script.js:32]: Yup, the whole public file is actually not needed because the js files are in the webapp folder

**Review by nlumiere — COMMENTED** [review]
lgtm!

**Review by arringtonh — COMMENTED** [review]
lgtm

File: proj/src/main/webapp/static/chat.html [hunk 3]
```diff
@@ -2,50 +2,43 @@
...
+    <link rel="stylesheet" href="../style.css">
+    </head>
+    
+    <body onload="init()">
+        <div class="container screen chat-page">
             <div class="row">
             <div class="col" id="chat-screen"> 
                 <h1> Chat Room </h1>
                 <div class="chat-and-input"> 
                     <div class="converstation">
```
Comment (aokeke) [inline proj/src/main/webapp/static/chat.html]: converstation->conversation typo?

File: proj/src/main/webapp/static/chat.html [hunk 4]
```diff
@@ -2,50 +2,43 @@
...
-                        <ul id="chat-messages">
-                        <li>Coffee</li>
-                        <li>Tea</li>
-                        <li>Milk</li>
-                        </ul>
+                        <ul id="chat-as-list">
```
Comment (aokeke) [inline proj/src/main/webapp/static/chat.html:34]: Does this need a closing tag? Also is it possible to include a username with the message? Would this depend on things that are not yet ready? If so, we can save that addition for later.

**Review by aokeke — CHANGES_REQUESTED** [review]

File: proj/src/main/webapp/style.css [hunk 5]
```diff
@@ -228,50 +222,49 @@ footer a:hover {
...
-.message {
-  border: thin solid lightgrey;
-  display: flex;
-  margin: 10px;
-  padding: 10px;
-  width: 600px;
-}
 
-.message {
+.converstation li {
```
Comment (showermat) [inline proj/src/main/webapp/style.css]: "Converstation"?

**Review by showermat — APPROVED** [review]

File: proj/src/main/webapp/static/chat.html (hunk 4 above)
Comment (leeannachen) [reply proj/src/main/webapp/static/chat.html:34]: Arrington actually addressed that in another pull request.  The list is changed into a div.

Comment (aokeke) [reply proj/src/main/webapp/static/chat.html:34]: Depending on who is closer to submitting their code first, you might want to pull that change into this branch to handle merge conflicts ahead of time.

Comment (leeannachen) [reply proj/src/main/webapp/static/chat.html:34]: good idea; i can wait for her to merge with master and i will pull from there

Comment (aokeke) [reply proj/src/main/webapp/static/chat.html:34]: If this one is ready first, you can also just merge this and then work with her to resolve merge conflicts on her PR as long as you both are in agreement, either is fine.

**Review by aokeke — APPROVED** [review]
LGTM

File: proj/src/main/webapp/static/chat.html (hunk 4 above)
Comment (leeannachen) [reply proj/src/main/webapp/static/chat.html:34]: gotcha

**Pull request merged**